### PR TITLE
[MRG+1] : add command to show meas info from the command line

### PR DESCRIPTION
**Pull request by agramfort**

### CODE REVIEW

File: mne/commands/tests/test_commands.py [hunk 1]
```diff
@@ -242,4 +243,8 @@ def test_flash_bem():
     os.chdir(currdir)
 
 
+def test_show_info():
+    """Test mne surf2bem"""
```
Comment (jaeilepp) [inline mne/commands/tests/test_commands.py]: Update comment.

**Comment by jaeilepp** [comment]
Could we also have a ``mne_show_fiff`` that takes an instance of raw instead of a filename?

**Comment by agramfort** [comment]
pushed

File: mne/commands/tests/test_commands.py [hunk 2]
```diff
@@ -242,4 +243,8 @@ def test_flash_bem():
     os.chdir(currdir)
 
 
+def test_show_info():
+    """Test mne show_info"""
+    check_usage(mne_show_info)
```
Comment (Eric89GXL) [inline mne/commands/tests/test_commands.py]: `show_info` is fast enough that you can actually check that it runs on some files. Don't get lazy on us :)

**Comment by agramfort** [comment]
@Eric89GXL done

ready for merge

**Comment by Eric89GXL** [comment]
+1 for merge from me once the CIs come back happy

**Comment by agramfort** [comment]
@Eric89GXL can you live with the with -0.2% coverage? or any idea how to fix it?

**Comment by Eric89GXL** [comment]
Nah I'll take a look

**Comment by Eric89GXL** [comment]
It says something changed in EOG artifact detection... doesn't make sense. Okay from my end.

**Comment by agramfort** [comment]
green button is all yours :)

**Pull request merged**